pkg/controller/certificates/trigger/trigger_controller.go

```diff
@@ -199,20 +199,37 @@ func (c *controller) ProcessItem(ctx context.Context, key string) error {
 	return nil
 }
 
-// shouldBackoffReissuingOnFailure tells us if we should back off from
-// reissuing the certificate and for how much time.
-func shouldBackoffReissuingOnFailure(log logr.Logger, c clock.Clock, crt *cmapi.Certificate) (backoff bool, delay time.Duration) {
+// shouldBackoffReissuingOnFailure tells us if this certificate's
+// re-issuance should be backed off and for how much time.
+//
+// The request can be left nil, in which case no back off is required. When
+// the certificate doesn't match the request, no back off is required
+// either since it means the request will have to be re-issued.
```

Comment:
The `either` here parses poorly.

I wonder if dropping the `,` would fix it.

Does this match what you're trying to say:
```suggestion
// The request can be left nil, in which case no back off is required.
// Additionally, if the certificate doesn't match the request, no back
// off is required because it means the request will have to be re-issued.
```

(I could be totally mis-parsing...)

Reply:
Agreed, this is poor phrasing on my side 😇